intTests/test_hash_table/Makefile

```
@@ -0,0 +1,3 @@
+ht_simplified.bc : ht_simplified.c
+	clang -c -emit-llvm -o ht_simplified.bc ht_simplified.c
```

Slightly terser:

```suggestion
ht_simplified.bc : ht_simplified.c
	clang -c -emit-llvm -o $@ $<
```

Also, consider adding the `-frecord-command-line` flag here so that the `.bc` file records precisely how it was compiled (including information about the Clang version).